README.md

```diff
@@ -12,16 +12,22 @@ Add to your serverless.yml:
 
   custom:
     assets:
-      bucket: my-bucket
-      files:
-        - source: ../assets/
-          globs: **/*.css
-        - source: ../app/
-          globs:
-            - **/*.js
-            - **/*.map
+       - bucket: my-bucket
```

Comment:
Yeah... strictly speaking, the ACL example would still work but I think I've just made it confusing. I'll go ahead and edit it.